src/libutil/util.hh

```diff
@@ -649,5 +649,5 @@ template<class... Ts> overloaded(Ts...) -> overloaded<Ts...>;
 
 std::string showBytes(uint64_t bytes);
 
-
+const std::string_view removeStartingOverlap(std::string & a_ref, std::string & b_ref);
```

Comment:
```suggestion
const std::string_view removeStartingOverlap(const std::string & a_ref, const std::string & b_ref);
```

(or directly take `string_view`s as input, C++ will happily do the conversion implicitely)